static/app/components/stream/group.tsx

```diff
@@ -136,9 +136,10 @@ function BaseGroupRow({
     };
   }, [organization, group.id, group.owners, query]);
 
-  const trackAssign: React.ComponentProps<typeof DeprecatedAssigneeSelector>['onAssign'] =
+  const updateGroup: React.ComponentProps<typeof AssigneeSelectorDropdown>['onAssign'] =
     useCallback(
-      (type, _assignee, suggestedAssignee) => {
+      async (type, assignee, suggestedAssignee) => {
+        await GroupStore.onAssignToSuccess(group.id, assignee.id, suggestedAssignee);
```

Comment:
The dropdown component now expects the parent component to handle updating the group upon an assignment change, hence the update to this onAssign callback